chemprop/cli/utils/parsing.py

```diff
@@ -163,9 +163,10 @@ def make_datapoints(
         the number of extra atom descriptors used for the j-th molecules. Any of the ``j`` lists can
         be a list of None values if the corresponding component does not use extra atom features. If
         ``None``, ``V_d`` for all datapoints will be ``None``.
-    features_generators : list[MoleculeFeaturizer] | None
-        a list of :class:`MoleculeFeaturizer` instances to generate additional molecule features to
-        use as extra descriptors
+    molecule_featurizers : list[str] | None
+        a list of molecule featurizer names to generate additional molecule features to use as extra
+        descriptors. If there are multiple molecules per datapoint, the featurizers will be applied
+        to each molecule and concatenated. A reaction has two molecules, reactant and product.
```

Comment:
Why "A reaction has two molecules, reactant and product." is mentioned here?

Reply:
I want users to understand that if they give a molecule_featurizer for a reaction datapoint, the featurizer will be applied to both the reactant and the product and then concatenated together. The extra descriptors for a reaction will therefore be twice as long as the extra descriptors for a molecule. 

Maybe it would be more clear if I said "Note that a reaction has two molecules, reactant and product." What do you think?

Reply:
I think it is great to keep this message. However, I think for a single reaction, we are not forced to consider the reactant and product as a single molecule; they can also be a complex (i.e., separated by a dot in SMILES representation)

Reply:
I think we're getting our semantics mixed between what a "molecule" means physically and symbolically (i.e., in RDKit). @KnathanM is referring to the latter and @shihchengli is referring former. Perhaps:

```suggestion
        to each molecule and concatenated. Note that a "reaction" logically corresponds to two RDKit :class:`~rdkit.Chem.Mol` objects, reactant and product, and each ``molecule_featurizer`` will be applied to both of these objects.
```

Reply:
I indeed had my semantics mixed up. Hopefully the updated string is clearer.